Sources/momento/CacheClient.swift

```diff
@@ -176,4 +177,428 @@ public class CacheClient: CacheClientProtocol {
         }
         return await self.dataClient.set(cacheName: cacheName, key: key, value: value, ttl: ttl)
     }
+    
+    /**
+     Adds multiple elements to the back of the given list. Creates the list if it does not already exist.
+     - Parameters:
+        - cacheName: the name of the cache to store the list in
+        - listName: the list to add to
+        - values: the elements to add to the list
+        - truncateFrontToSize: If the list exceeds this length, remove excess from the front of the list. Must be positive.
+        - ttl: refreshes the list's TTL using the client's default if this is not supplied.
+     - Returns: CacheListConcatenateBackResponse representing the result of the operation.
+     Pattern matching can be used to operate on the appropriate subtype.
+    ```
+     switch response {
+     case let responseError as CacheListConcatenateBackError:
+        // handle error
+     case let responseSuccess as CacheListConcatenateBackSuccess:
+        // handle success
+     }
+    ```
+     */
+    public func listConcatenateBack(
+        cacheName: String,
+        listName: String,
+        values: [ScalarType],
+        truncateFrontToSize: Int? = nil,
+        ttl: TimeInterval? = nil
```

Comment:
The collection data type opertions handle TTLs differently from the scalar operations, using something we've termed a `CollectionTtl`. In addition to a duration, this object/struct also contains a boolean called `refreshTtl` that allows users to manipulate collections without modifying the collections' ttls if desired. We're going to need to add this struct and use it here as well.

Here's a [golang example](https://github.com/momentohq/client-sdk-go/blob/main/utils/collection_ttl.go) and a [php example](https://github.com/momentohq/client-sdk-php/blob/main/src/Requests/CollectionTtl.php) of the type. You can check out how the CollectionTtl is processed before the request [in this PHP list function](https://github.com/momentohq/client-sdk-php/blob/0cb716e5f8a9c7d0f6bb29ac332363bbee8fc6fc/src/Cache/Internal/ScsDataClient.php#L503)